.github/workflows/ci.yaml

```diff
@@ -33,6 +33,12 @@ jobs:
 
     - run: nix-shell --run 'build-ci.sh'
 
+    - name: Publish Unit Test Results
+      uses: EnricoMi/publish-unit-test-result-action@v1
+      if: always()
```

Comment:
Hm.. I have no idea of github action but this looks weird :sweat_smile:

Reply:
I think you're speaking about the `always`? I guess it's there because the same action can be triggered by various events. I will remove it.